pymc/sampling_jax.py

```diff
@@ -264,5 +264,6 @@ def sample_numpyro_nuts(
         dims=dims,
         **idata_kwargs,
     )
+    az_trace.sample_stats.attrs["sampling_time"] = (tic3 - tic2).total_seconds()
```

Comment:
Compare [this line](https://github.com/pymc-devs/pymc/blob/main/pymc/backends/arviz.py#L155). Attributes should be provided via the `attrs` kwarg.
```suggestion
        attrs={
            "sampling_time": (tic3 - tic2).total_seconds(),
        },
        **idata_kwargs,
    )
    az_trace.sample_stats.attrs["sampling_time"] = (tic3 - tic2).total_seconds()
```
<sup>This suggestion does not come with a `black` guarantee.</sup>

Reply:
Changed!